### ISSUE TITLE
Support for Echo Show 8?

### ISSUE THREAD

**Issue by andre2007** [datech/node-red-contrib-amazon-echo#107]
As my current Alexa device does not have the feature to discover local devices, I think about buying another echo device. Echo Show 8 looks quite interesting but isn't supported by your plugin. Do you plan to add support also for this device in future?

**Comment by andre2007** [comment]
I just noticed, it is possible to install Alexa on a Raspberry Pi. Is it possible that your plugin supports Alexa running on a Raspberry Pi?

**Comment by datech** [comment]
IFAIK AVS will not be working. You may check #53

Echo Show 8 might be working, but no one confirmed this. That's why it is not included in the list with supported devices.

**Comment by andre2007** [comment]
I tried it with `Echo Show 2. Gen`. Node-Red is running as plugin within Home Assistant. 

I deployed a flow with an echo hub and an echo device. Alexa app is able to find the device as light bulb.
Also Home Assistant shows the dialog to configure the new device `Philips Hue` but here I stuck. The HA dialog says I should press a physical button on the Hue Hub device. The hue hub is an embedded functionality of the `echo ahow` and I do not have such a button.

If I just press the submit button in HA there is an error message `Unknown link error`.

Should it be possible to configure the virtual device within HA or it this error expected?
